app/schemas/email.py

```
@@ -0,0 +1,125 @@
+    )
+    
+    model_config = ConfigDict(populate_by_name=True)
+
+    @field_validator(
+        "subject",
+        "from_address",
+        "envelope_from",
+        "reply_to",
+    )
+    @classmethod
+    def _reject_crlf_in_string_fields(cls, value):
```

Ruff (0.14.10)

Missing return type annotation for classmethod `_reject_crlf_in_string_fields`

(ANN206)